src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/leaked-passwords/[type]/page.tsx

```diff
@@ -51,15 +55,12 @@ export default async function LeakedPasswords({
       subscriberEmails={subscriberEmails}
       type={type}
       data={{
-        countryCode: getCountryCode(headers()),
+        countryCode: countryCode,
```

Comment:
```suggestion
        countryCode,
```

Reply:
Removed redundant key/value pairs in [dd3200d](https://github.com/mozilla/blurts-server/pull/4212/commits/dd3200d38555c0993d42adb94d364cff3670b585).